Makefile

```diff
@@ -233,18 +233,21 @@ dist/argoexec-%: $(ARGOEXEC_PKGS)
 executor-image: $(EXECUTOR_IMAGE_FILE)
 
 	# Create executor image
-$(EXECUTOR_IMAGE_FILE):
+$(EXECUTOR_IMAGE_FILE): $(ARGOEXEC_PKGS)
 	$(call docker_build,argoexec,argoexec,$(EXECUTOR_IMAGE_FILE))
 
 # generation
 
-$(HOME)/go/bin/mockery:
-	$(call backup_go_mod)
-	go get github.com/vektra/mockery/.../
-	$(call restore_go_mod)
+$(GOPATH)/bin/mockery:
+	./hack/recurl.sh dist/mockery.tar.gz https://github.com/vektra/mockery/releases/download/v1.1.1/mockery_1.1.1_$(shell uname -s)_$(shell uname -m).tar.gz
+	tar zxvf dist/mockery.tar.gz mockery
+	chmod +x mockery
+	mkdir -p $(GOPATH)/bin
+	mv mockery $(GOPATH)/bin/mockery
```

Comment:
1. Manually overwriting binaries in `go/bin` might break some existing users' flow with `go get`. The download is 2 MB... perhaps we should keep in in the `dist` folder?

1. What happens if the file already exists?

Reply:
if the file already exists, nothing happens, we want to install it permanently, we could go back to `go get`?

Reply:
reason to install permanently = reliability + speed

Reply:
> if the file already exists, nothing happens

This is good enough, then. IMO